packages/framework/autoload-easy-coding-standard.yml

```diff
@@ -27,3 +27,14 @@ parameters:
         PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\EmptyStatementSniff.DetectedWhile:
             - '*/src/Model/Product/Availability/ProductAvailabilityRecalculator.php'
             - '*/src/Model/Product/Pricing/ProductPriceRecalculator.php'
+
+        Shopsys\CodingStandards\Sniffs\ObjectIsCreatedByFactorySniff:
+          - '*/tests/*'
+          - '*DataFixture.php'
```

Comment:
I do not like all the other exceptions, are they justified? Would it be difficult to fix the violations of the rule? If not, please do that. Otherwise you should explain all the exceptions here by adding comments and create a new user story for resolving it.

Reply:
ok, but even in data fixtures, the instances should be created via factory - I have added it in the issue, please move the comment in the code right after the line with - '*/tests/*'